src/main/java/io/ebean/config/ServerConfig.java

```diff
@@ -2985,6 +2995,13 @@ protected void loadSettings(PropertiesWrapper p) {
 
     String mappingsProp = p.get("mappingLocations", null);
     mappingLocations = getSearchList(mappingsProp, mappingLocations);
+
+    if (!p.getBoolean("failOnLoadError", true)) {
```

Comment:
failOnLoadJsonError ?

Reply:
It does not affect only to json errors. It also may handle also unparseable Durations and so on, that were stored as String